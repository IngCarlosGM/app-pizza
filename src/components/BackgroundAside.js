import React from 'react';

const BackgroundAside = () => {

    return (
        <div className="relative">
            <img 
                className="object-fit"
                src={`../assets/imagebkg.png`} 
                alt="Fondo negro de la ilustraciÃ³n" 
            />

            <img 
                className="center-vertical-horizontal-absolute size-image-pizza absolute"
                src={`../assets/Pizza.png`} 
                alt="Imagen de una Pizza"
            />

            <img 
                className="absolute logo-aside"
                src={`../assets/Logo.png`} 
                alt="Logo de Best pizza"
            />
        </div>
    )
}

export default BackgroundAside;
