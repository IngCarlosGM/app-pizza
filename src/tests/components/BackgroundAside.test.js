import { shallow } from "enzyme";
import BackgroundAside from "../../components/BackgroundAside";

describe('Teting DescriptionPage component', () => {

    const wrapper = shallow(
        <BackgroundAside />
    )
   
    test('should render correctly', () => {
        expect( wrapper ).toMatchSnapshot();
    });

    test('should to have a src equal to ../assets/imagebkg.png', () => {
        expect( wrapper.find('img').at(0).prop('src') ).toBe( '../assets/imagebkg.png' );
    });

    test('should to have a src equal to ../assets/Pizza.png', () => {
        expect( wrapper.find('img').at(1).prop('src') ).toBe( '../assets/Pizza.png' );
    });

    test('should to have a src equal to ../assets/Logo.png', () => {
        expect( wrapper.find('img').at(2).prop('src') ).toBe( '../assets/Logo.png' );
    });
    
});
