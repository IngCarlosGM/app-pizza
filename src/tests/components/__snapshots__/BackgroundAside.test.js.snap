// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teting DescriptionPage component should render correctly 1`] = `
<aside
  className="relative"
>
  <img
    alt="Fondo negro de la ilustraciÃ³n"
    className="object-fit"
    src="../assets/imagebkg.png"
  />
  <img
    alt="Imagen de una Pizza"
    className="center-vertical-horizontal-absolute size-image-pizza absolute img-responsive "
    src="../assets/Pizza.png"
  />
  <img
    alt="Logo de Best pizza"
    className="absolute logo-aside img-responsive "
    id="logo-best-pizza-aside"
    src="../assets/Logo.png"
  />
</aside>
`;
